debian/control

```diff
@@ -13,9 +13,9 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, python3, python3-pyqt5, python3-pyq
 Description: securedrop client for qubes workstation
 
 Package: securedrop-export
-Architecture: all
-Depends: ${misc:Depends}, python3, udisks2, cups, printer-driver-brlaser, printer-driver-hpcups, system-config-printer, xpp, libcups2, gnome-disk-utility, libreoffice,
- desktop-file-utils, shared-mime-info, libfile-mimeinfo-perl
+Architecture: amd64
```

Comment:
I don't think you need to change the architecture away from `all`, as far as I can tell, there's no compiled code being shipped in the export package.